### ISSUE TITLE
UITableViewAlertForLayoutOutsideViewHierarchy

### ISSUE THREAD

**Issue by paulz** [inamiy/Harvest-SwiftUI-Gallery#16]
Within few seconds app generates over 1000 state changes while the time is running.

Using Stopwatch cause app to freeze with the warning in the console:
UITableViewAlertForLayoutOutsideViewHierarchy

```
2020-01-26 14:18:28.959161-0800 Harvest-SwiftUI-Gallery[62071:3626159] [TableView] 
Warning once only: UITableView was told to layout its visible cells and other contents without
 being in the view hierarchy (the table view or one of its superviews has not been added to a 
window). This may cause bugs by forcing views inside the table view to load and perform layout 
without accurate information (e.g. table view bounds, trait collection, layout margins, safe area 
insets, etc), and will also cause unnecessary performance overhead due to extra layout passes. 
Make a symbolic breakpoint at UITableViewAlertForLayoutOutsideViewHierarchy to catch this in 
the debugger and see what caused this to occur, so you can avoid this action altogether if 
possible, or defer it until the table view has been added to a window. Table view: 
<_TtC7SwiftUIP33_BFB370BA5F1BADDC9D83021565761A4925UpdateCoalescingTableView: 0x7fbd1b127000; baseClass = UITableView; frame = (0 0; 375 320.667); clipsToBounds = YES; autoresize = W+H; gestureRecognizers = <NSArray: 0x60000300f5d0>; layer = <CALayer: 0x600003ed6a00>; contentOffset: {0, 0}; contentSize: {375, 132.9999974568685}; adjustedContentInset: {0, 0, 0, 0}; dataSource: <_TtGC7SwiftUIP10$10f027a4419ListCoreCoordinatorGVS_20SystemListDataSourceOs5Never_GOS_19SelectionManagerBoxS2___: 0x7fbd1976bbf0>>

```
![Simulator Screen Shot - iPhone 11 Pro - 2020-01-26 at 14 20 30](https://user-images.githubusercontent.com/59230/73142733-06d99580-4047-11ea-92e6-9d34a45a7ad1.png)

Xcode: Version 11.3.1 (11C504)

**Comment by inamiy** [comment]
Probably the similar issue as https://github.com/inamiy/Harvest-SwiftUI-Gallery/issues/14#issuecomment-619478666 where running Stopwatch (heavy task) causes related bugs.

**Comment by inamiy** [comment]
#20 (turning off TimeTravel debugging feature) will hopefully fix many UI glitches reported in #14, #15, #16 .

Please let me know if the same issues still occur.
